srcpkgs/NetworkManager/patches/fix-cross-gir.patch

```diff
@@ -1,10 +1,10 @@
-Reason: Fix crossbuilding GIR
-Upstream: No, not upstreamable since this is very specific to the Void setup
-Index: b/src/libnm-client-impl/meson.build
-===================================================================
+# Reason: Fix crossbuilding GIR
+# Upstream: No, not upstreamable since this is very specific to the Void setup
+diff --git a/src/libnm-client-impl/meson.build b/src/libnm-client-impl/meson.build
+index 46464a6328..27a42f51e6 100644
 --- a/src/libnm-client-impl/meson.build
 +++ b/src/libnm-client-impl/meson.build
```

Comment:
what's the reason for this?

Reply:
The patch no longer applied due to the meson.build file changing slightly, so I decided to recreate it. That said, the docs option now requires the introspection option, so I'm wondering if there may be a way to use `vopt` to drop the need for this patch altogether.

Reply:
this patch applies fine, the other one needed a rebase